<!-- // 2024 Rebuttal -->
<launch>
    <node name="ctrl_log" pkg="nics_visualization" type="ctrl_log" output="screen">
        <param name="/log_path" type="string" value="/home/patty/nics_ws/src/NICS/nics_visualization/logs/thesis/"/>
        <param name="/filename1" type="string" value="0821_ctrl_tracking_B.csv"/>
    </node>         
</launch>
